src/main/java/ee/tuleva/onboarding/auth/ocsp/OCSPRequest.java

```diff
@@ -0,0 +1,155 @@
+package ee.tuleva.onboarding.auth.ocsp;
+
+import com.codeborne.security.AuthenticationException;
```

Comment:
Maybe we should define our own `AuthenticationException` class instead. Otherwise we're dependent on the `mobile-id` library.